.github/workflows/publish.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-python@v1
+      with:
+        python-version: '3.7'
```

Comment:
I'm okay with Python 3.7, but just wondering if there's a reason why it can't be 3.8 or 3.9?

Reply:
No particular reason. This is a straight copy of the publish action in the core repo